compat/src/render.js

```diff
@@ -52,12 +52,12 @@ options.event = e => {
 	if (oldEventHook) e = oldEventHook(e);
 	e.persist = () => {};
 	e.isDefaultPrevented = () => e.defaultPrevented;
-	const normalStopPropagation = e.stopPropagation;
 	let stoppedPropagating = false;
-	e.stopPropagation = function() {
-		stoppedPropagating = true;
-		normalStopPropagation.call(this);
-	};
+	Object.defineProperty(e, 'stopPropagation', {
+		value() {
+			stoppedPropagating = true;
```

Comment:
Looking at [the page in mdn](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) I'm wondering if we still need to call the original method somehow.

Reply:
Why, we just add an invocation listener with this?